scipy/signal/signaltools.py

```diff
@@ -302,6 +262,73 @@ def correlate(in1, in2, mode='full', method='auto', return_lags=False):
                          " 'direct', or 'fft'.")
 
 
+def correlation_lags(in1, in2, mode='full'):
```

Comment:
@peterbell10 is this what you had in mind?

Reply:
Yes, I think this makes more sense.

Although, Instead of taking `in1` and `in2`, I would take integers `len1` and `len2`. Then the user can easily use this function for multidimensional arrays:

```
axis_lags = correlation_lags(in1.shape[axis], in2.shape[axis], mode=mode)
```